source/fundamentals/crud/write-operations/update.txt

```diff
@@ -425,11 +426,45 @@ with a new ``Restaurant`` document named "Food World". Because the ``IsUpsert``
 set to ``true``, the driver inserts a new document if the query filter doesn't 
 match any existing documents.
 
-.. literalinclude:: /includes/fundamentals/code-examples/crud/change/Replace.cs
-   :language: csharp
-   :dedent:
-   :start-after: // start-method-body
-   :end-before: // end-method-body
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: csharp
+
+      var filter = Builders<Restaurant>.Filter.Eq(r => r.Name, "Pizza Town");
+
```

Comment:
[suggestion]
Similar to my comment in the code - perhaps consider changing this `r` to `restaurant` to match the other code examples and also improve understanding?